### ISSUE TITLE
/root/miniconda2/envs/snake27.10/lib/python2.7/site-packages/bokeh/util/deprecation.py:34: BokehDeprecationWarning: bokeh.util.notebook was deprecated in Bokeh 0.12.10 and will be removed, use bokeh.io.notebook instead.   warn(message)

### ISSUE THREAD

**Issue by jmbertoncelli** [ibm-watson-data-lab/pixiedust#499]
Hello since my last conda --upade --all I get this warning....

Pixiedust database opened successfully
 Pixiedust version 1.1.1
/root/miniconda2/envs/snake27.10/lib/python2.7/site-packages/bokeh/util/deprecation.py:34: BokehDeprecationWarning: bokeh.util.notebook was deprecated in Bokeh 0.12.10 and will be removed, use bokeh.io.notebook instead.
  warn(message)

**Comment by DTAIEB** [comment]
@jmbertoncelli yes, we've started seeing this and other issues since bokeh 0.12.6. We're working on a comprehensive fix. For now, please lock your version of Bokeh to 0.12.6
